### PR TITLE
[BTAT-8948] Changes to view for charity change of org name

### PR DESCRIPTION
**Pull request by daparkes**
# Pull Request Checklist
* [x] Branch is rebased with master
* [x] Added Unit Tests for changed functionality
* [x] Ran `sbt clean compile coverage test it:test coverageReport`, all tests pass and coverage meets minimum
* [x] Checked libraries are up to date with `sbt validate`. (If applicable) library updates are done in a separate, single commit
* [x] Ran Acceptance tests as per the story. If any query ping in group slack and confirm the AT's repo
* [x] Ran Performance tests as per the story. If any query ping in group slack and confirm the PT's repo
* [x] (If applicable) raised app-config-* PRs
* [x] (If applicable) ran Wave test for changes to views
* [x] (If applicable) added Integration Test

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -152,6 +152,7 @@ auditing {
 party-types = ["7","50"]
 party-types-r19 = ["Z1","1","2","3","4","5","6","7","8","9","10","50","51","52","53","54","55","58","59","60","61","62","63"]
 party-types-nspItmpOrSAMastered = ["Z1", "1", "2", "3", "6", "52", "55", "58", "59", "61", "63"]
+party-types-trusts = ["6", "60"]
```
Comment (lewishay) [inline conf/application.conf:155]: I can see party type 6 is also in the list above. I think we need to understand which route party type 6 should go down.

File: app/views/businessName/AltChangeBusinessNameView.scala.html [hunk 2]
```diff
@@ -19,33 +19,41 @@
 
 @this(mainTemplate: MainTemplate, headings: Headings, backToBusinessDetails: BackToBusinessDetails)
 
-@(organisationName: String)(implicit user : User[_], messages: Messages, appConfig: config.AppConfig)
+@(organisationName: String, orgIsTrust: Boolean)(implicit user : User[_], messages: Messages, appConfig: config.AppConfig)
```
Comment (lewishay) [inline app/views/businessName/AltChangeBusinessNameView.scala.html:22]: I think instead of having a boolean here (and potentially extra booleans for any other cases going forward), the best way to make use of a dynamic view like this is for the controller to pass in a view model which contains the dynamic components. Then the view can be stay simple as it just pulls the parts out of the model. It also helps generally to have less Scala logic in a view where possible.

So to summarise - could this be refactored so that the controller does the logic and the view simply accepts a view model?

**Review by lewishay — CHANGES_REQUESTED** [review]
